bin/node-template/node/Cargo.toml

```diff
@@ -17,7 +17,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 name = "node-template"
 
 [dependencies]
-clap = { version = "3.1.18", features = ["derive"] }
+clap = { version = "4.0.9", features = ["derive", "deprecated"] }
```

Comment:
This prints warnings for breaking changes in the next release. We used in on cumulus after I upgraded to 3.2.22 to make people aware that they should not use the old API. But we want to get rid of it here since it could lead to new warnings if the clap version changes.